samples/common.sh

```diff
@@ -0,0 +1,20 @@
+samples_dir=$(dirname $(realpath "$BASH_SOURCE"))
+SGXLKL_ROOT=$(realpath "${samples_dir}/..")
+
+if [[ -z "${SGXLKL_PREFIX}" ]]; then
```

Comment:
These are already exported in Makefile. Do we need to export again? Also SGXLKL_SETUP_TOOL exists in common.sh and not in common.mk just FYI

Reply:
Some samples need this in the bash script because not all of the commands are run in the Makefile. For example, ``SGXLKL_SETUP_TOOL`` is needed in ``containers/redis/test.sh``. A couple of them are needed in ``languages/python/test.sh``.

I only added ``SGXLKL_SETUP_TOOL`` to the bash script because it's not needed in the Makefiles.